scripts/gulpfiles/build_tasks.js

```diff
@@ -630,6 +630,15 @@ function buildCompiled() {
     // option to Closure Compiler; instead feed them as input via gulp.src.
   };
 
+  // Symlink source dirs from build dir so that sourcemaps work in
+  // compiled-mode testing.
+  for (const src of ['core', 'blocks', 'generators']) {
+    const target = `${BUILD_DIR}/${src}`
+    if (!fs.existsSync(target)) {
+      fs.symlinkSync(`../${dir}`, target);
```

Comment:
This doesn't build, should it be `src`? if so then lgtm

Reply:
Oh weird.  I fixed this locally and had thought I had done a force-push _before_ opening the PR, but evidently I forgot to `git add` the change before committing.  Oops!